finagle-core/src/main/scala/com/twitter/finagle/Dtabs.scala

```diff
@@ -0,0 +1,13 @@
+package com.twitter.finagle
+
+/**
+ * Static methods for Java callers

```

Comment:
I love that you're thinking of java users here!  But shouldn't java-users be able to use static forwarders on Dtab?  If they can't, why can't they?

Reply:
The java compilation test started failing, so I just did what I had to do. I assumed that the underlying cause there are no static forwarders anymore is because we go through a common trait now, based on the flimsiest of intuition. I'll do some experiments to find the cause, and see what bytecode is generated under what conditions.

Reply:
I'd rather keep those static forwarders if we can.  might be worth duplicating more code, or else making DtabBase an abstract class.

Reply:
I'll toy with it a bit. I'll get them back one way or another.

Reply:
I thought I broke my local cache with that last one. I either broke scalac or sbt. Oh well.

Reply:
thanks!